src/components/Onboarding/OnboardingCarousel.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import Slider, { Settings } from 'react-slick';
+import styled from '@emotion/styled';
+import image_1 from '@assets/planet_main.svg';
+import image_2 from '@assets/onboarding_2.svg';
+import image_3 from '@assets/onboarding_3.svg';
+import CarouselPage, { CarouselContent } from './CarouselPage';
+
+interface CarouselProps {
+  updateSlideIndex: React.Dispatch<React.SetStateAction<number>>;
+}
+
+function OnboardingCarousel({ updateSlideIndex }: CarouselProps) {
+  const pages: CarouselContent[] = [
+    {
+      id: 0,
+      title: '기록 행성',
+      imageUrl: image_1,
+      description: '행성에 들어가 감정을 선택하고\n글을 작성해보세요.',
+    },
+    {
+      id: 1,
+      title: '감정 행성',
+      imageUrl: image_2,
+      description: '감정 행성에 들어가 별을 누르면\n기록을 볼 수 있어요.',
+    },
+    {
+      id: 2,
+      title: '별자리',
+      imageUrl: image_3,
+      description: '감정 기록을 많이 모을수록\n별자리가 채워져요.',
+    },
+  ];
+
+  const sliderProps: Settings = {
+    arrows: true,
+    infinite: false,
+    autoplay: false,
+    pauseOnHover: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    initialSlide: 0,
+  };
+
+  const [didScroll, SetDidScroll] = useState<boolean>(false);
+
+  const handleBeforeChange = (beforeSlide) => {
+    didScroll || updateSlideIndex(beforeSlide + 1);
+  };
+
+  const handleAfterChange = (currentSlide) => {
+    SetDidScroll(true);
+    updateSlideIndex(currentSlide);
+  };
```

Comment:
이렇게만 해도 정상 동작하는 것 같은데, 혹시 didScroll과 afterChange는 왜 필요할까요?

```suggestion
  const handleBeforeChange = (beforeSlide) => {
    updateSlideIndex(beforeSlide + 1);
  };

```